packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/local.tsx

```diff
@@ -1,50 +1,21 @@
 import { Dialog, Icon, MobileWrapper, Text } from '@deriv/components';
 import { isMobile } from '@deriv/shared';
 import { Localize, localize } from '@deriv/translations';
+import { observer } from '@deriv/stores';
 import classNames from 'classnames';
 import { DBOT_TABS } from 'Constants/bot-contents';
 import { clearInjectionDiv } from 'Constants/load-modal';
 import React from 'react';
-import { connect } from 'Stores/connect';
-import RootStore from 'Stores/index';
+import { useDBotStore } from 'Stores/useDBotStore';
 import BotPreview from './bot-preview';
 import './index.scss';
 
-type TWorkspace = {
-    id: string;
-    xml: string;
-    name: string;
-    timestamp: number;
-    save_type: string;
-};
+const LocalComponent = observer(() => {
+    const { load_modal, save_modal, dashboard } = useDBotStore();
+    const { handleFileChange, loadFileFromRecent, dashboard_strategies } = load_modal;
+    const { onConfirmSave } = save_modal;
+    const { setActiveTab, setPreviewOnDialog, has_mobile_preview_loaded } = dashboard;
 
-type Nullable<T> = T | null;
-type TLocalComponent = {
-    handleFileChange: (e: React.ChangeEvent<HTMLInputElement>, data: boolean) => boolean;
-    loadFileFromRecent: () => void;
-    onConfirmSave: () => void;
-    onDrop: () => void;
-    handleTabChange: (param: number) => void;
-    previewRecentStrategy: () => void;
-    setActiveTab: (param: number) => void;
-    dashboard_strategies: Array<TWorkspace>;
-    setFileLoaded: (param: boolean) => void;
-    setLoadedLocalFile: (data: Nullable<string>) => void;
-    setTourDialogVisibility: (param: boolean) => boolean;
-    setPreviewOnDialog: (param: boolean) => boolean;
-    has_mobile_preview_loaded: boolean;
-};
```

Comment:
@vinu-deriv if we removed types of props and didn't add these types to types of the store, then it means we already have types in the store, haven't it?

Reply:
we already have types in `@deriv/stores`